go.mod

```diff
@@ -16,50 +16,24 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.2
 	github.com/google/btree v1.0.0
+	github.com/gopherjs/gopherjs v0.0.0-20190812055157-5d271430af9f // indirect
```

Comment:
?

Reply:
```
This module is necessary because github.com/gopherjs/gopherjs/js is imported in:

- github.com/celestiaorg/celestia-core/cmd/tendermint/commands
-- github.com/spf13/viper
--- gopkg.in/ini.v1
---- gopkg.in/ini.v1.test
----- github.com/smartystreets/goconvey/convey
------ github.com/jtolds/gls
```